OneCommunityGlobal/HGNRest
# Parthj unit and integration test forgotpwdcontroller.js

# Description
PR adds unit test for forgotPwdcontroller.js, integration test for forgotPwdRouter.js and a readme highlighting the test cases covered in both.

## Related PRS (if any):
None

## Main changes explained:
- Added unit test file named: forgotPwdcontroller.spec.js
- Added integration test file named: forgotPwdRouter.test.js

## How to test:
1. check into current branch
2. do `npm install` 
3. do `npm run test forgotPwd`
4. All test cases should pass

## Screenshots or videos of changes:
Output looks like this:
<img width="953" alt="Screenshot 2024-05-25 at 6 29 43 PM" src="https://github.com/OneCommunityGlobal/HGNRest/assets/56110372/07756d8d-f8a4-42e0-bf38-83baa1829b48">
